allennlp/tests/models/encoder_decoders/simple_seq2seq_test.py

```diff
@@ -18,26 +21,34 @@ def test_model_can_train_save_and_load(self):
         self.ensure_model_can_train_save_and_load(self.param_file, tolerance=1e-2)
 
     def test_bidirectional_model_can_train_save_and_load(self):
-        param_file = self.FIXTURES_ROOT / "encoder_decoder" / \
-                     "simple_seq2seq" / "experiment_bidirectional.json"
-        self.ensure_model_can_train_save_and_load(param_file, tolerance=1e-2)
+        param_overrides = json.dumps({"model": {"encoder": {"bidirectional": True}}})
+        self.ensure_model_can_train_save_and_load(self.param_file,
+                                                  tolerance=1e-2,
+                                                  overrides=param_overrides)
 
     def test_no_attention_model_can_train_save_and_load(self):
-        param_file = self.FIXTURES_ROOT / "encoder_decoder" / \
-                     "simple_seq2seq" / "experiment_no_attention.json"
-        self.ensure_model_can_train_save_and_load(param_file, tolerance=1e-2)
+        param_overrides = json.dumps({"model": {"attention": None}})
+        self.ensure_model_can_train_save_and_load(self.param_file,
+                                                  tolerance=1e-2,
+                                                  overrides=param_overrides)
 
     def test_legacy_attention_model_can_train_save_and_load(self):
-        param_file = self.FIXTURES_ROOT / "encoder_decoder" / \
-                     "simple_seq2seq" / "experiment_legacy_attention.json"
-        self.ensure_model_can_train_save_and_load(param_file, tolerance=1e-2)
+        param_overrides = json.dumps({"model": {
+                "attention": None,
+                "attention_function": {"type": "dot_product"}
+        }})
+        self.ensure_model_can_train_save_and_load(self.param_file,
+                                                  tolerance=1e-2,
+                                                  overrides=param_overrides)
 
     def test_greedy_model_can_train_save_and_load(self):
-        param_file = self.FIXTURES_ROOT / "encoder_decoder" / \
-                     "simple_seq2seq" / "experiment_no_beam_search.json"
-        self.ensure_model_can_train_save_and_load(param_file, tolerance=1e-2)
+        param_overrides = json.dumps({"model": {"beam_size": None}})
+        self.ensure_model_can_train_save_and_load(self.param_file,
+                                                  tolerance=1e-2,
+                                                  overrides=param_overrides)
 
     def test_loss_is_computed_correctly(self):
+        # pylint: disable=protected-access
```

Comment:
I'm going to merge this without worrying about it, but just for future reference, it's always fine to just put this at the top of the file in a test.  A bunch of our tests do this.